src/css/sass.js

```diff
@@ -114,8 +127,67 @@ export default class SassCompiler extends CompilerBase {
     };
   }
 
+  buildImporterCallback (includePaths) {
+    const self = this;
+    return (function (request, done) {
+      let file
+      if (request.file) {
+        done();
+      } else {
+        // sass.js works in the '/sass/' directory
+        const cleanedRequestPath = request.resolved.replace(/^\/sass\//, '');
+        for (let includePath of includePaths) {
+          const filePath = path.resolve(includePath, cleanedRequestPath);
+          let variations = sass.getPathVariations(filePath);
+
+          file = variations
+            .map(self.fixWindowsPath.bind(self))
+            .reduce(self.importedFileReducer.bind(self), null);
+
+          if (file) {
+            const content = fs.readFileSync(file, { encoding: 'utf8' });
+            return sass.writeFile(file, content, () => {
+              done({ path: file })
```

Comment:
return here too - even though we don't need it, just get in the habit of always writing return after a callback, it will save you much sanity in the long run